GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaHashset.java

```diff
@@ -0,0 +1,28 @@
+package hackkerank.day3;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class JavaHashset {
+
+    public static void main(String[] args) {
+
+        Scanner s = new Scanner(System.in);
+        int t = s.nextInt();
+        String[] pair_left = new String[t];
```

Comment:
snake case???
Java prefers CamelCase for var names
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case